modules/hello-world/examples/KvHelloWorldScoped.java

```diff
@@ -0,0 +1,224 @@
+
+      // Print the result's CAS metadata to the console.
+      System.out.println("CAS:" + getResult.cas());
+      System.out.println("Data:" + getResult.contentAsObject());
+      System.out.println("Expiry:" + getResult.expiryTime());
+      // end::kv-get-with-opts[]
+    }
+
+    {
+      System.out.println("Example: [kv-get-subdoc]");
+      // tag::kv-get-subdoc[]
+      List<LookupInSpec> specs = new ArrayList<>();
+      specs.add(LookupInSpec.get("geo"));
+
+      LookupInResult lookupInResult = hotelCollection.lookupIn("hotel-123", specs);
+      System.out.println("CAS:" + lookupInResult.cas());
+      System.out.println("Geo:" + lookupInResult.contentAsObject(0));
+      // end::kv-get-subdoc[]
+    }
+
+    {
+      System.out.println("Example: [kv-update-replace]");
+      // tag::kv-update-replace[]
+      // Fetch an existing hotel document
+      GetResult getResult = hotelCollection.get("hotel-123");
+      JsonObject existingDoc = getResult.contentAsObject();
+
+      // Get the current CAS value.
+      Long currentCas = getResult.cas();
+      System.out.println("Current CAS:" + currentCas);
+
+      // Add a new review to the reviews array.
+      existingDoc.getArray("reviews").add(JsonObject.create()
+          .put("content", "This hotel was cozy, conveniently located and clean.")
+          .put("author", "Carmella O'Keefe")
+          .put("date", DateTimeFormatter.ISO_INSTANT.format(Instant.now())
+        )
+      );
+
+      // Update the document with new data and pass the current CAS value. 
+      MutationResult replaceResult = hotelCollection.replace(
+          "hotel-123",
+          existingDoc,
+          ReplaceOptions.replaceOptions().cas(currentCas)
+      );
+
+      // Print the new CAS value.
+      System.out.println("New CAS:" + replaceResult.cas());
+      // end::kv-update-replace[]
+    }
+
+    {
+      System.out.println("Example: [kv-update-upsert]");
+      // Create the document object.
+      JsonObject document = JsonObject.create()
+          .put("id", "hotel-123")
+          .put("name", "Medway Youth Hostel")
+          .put("address", "Capstone Road, ME7 3JE")
+          .put("url", "http://www.yha.org.uk")
+          .put("country", "United Kingdom")
+          .put("city", "Medway")
+          .put("state", (String) null)
+          .put("vacancy", true)
+          .put("description", "40 bed summer hostel about 3 miles from Gillingham.");
+
+      // tag::kv-update-upsert[]
+      // Update or create a document in the hotel collection.
+      MutationResult upsertResult = hotelCollection.upsert("hotel-123", document);
+
+      // Print the result's CAS metadata to the console.
+      System.out.println("CAS:" + upsertResult.cas());
+      // end::kv-update-upsert[]
+    }
+
+    {
+      System.out.println("Example: [kv-update-subdoc]");
+      // tag::kv-update-subdoc[]
+      List<MutateInSpec> specs = new ArrayList<>();
+      specs.add(MutateInSpec.upsert("pets_ok", true));
```

Comment:
I've seen (and used) `= Arrays.asList(MutateInSpec.upsert(...))` elsewhere instead.

Reply:
Same here, I think I've forgotten to do it in these examples, 👍 I'll update.